src/components/Overlay/OverlayToolbar.tsx

```diff
@@ -66,7 +70,10 @@ OverlayToolbarClose.propTypes = {
   onClick: PropTypes.func.isRequired
 }
 
-export const OverlayToolbar = ({ title, onClose, children }) => {
+export const OverlayToolbar: React.FC<{
+  title: string | ReactNode
+  onClose: () => void
```

Comment:
Fine here, since it's abstracted

Reply:
I am not sure I get what you trying to say. I used `React.FC` since the component makes explicit use of `children` (if that's what you refer to).